src/Validator/LIVR/Rules/ListLength.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace uMotif\Validator\LIVR\Rules;
+
+final class ListLength
+{
+    const FORMAT_ERROR = 'FORMAT_ERROR';
+
+    public static function listLength($param1, $param2)
+    {
+        return function ($values) use ($param1, $param2) {
+            if (!isset($param1) || $param1 === '' || $param2 < $param1) {
+                return self::FORMAT_ERROR;
+            }
+
+            $minLen = $param1;
+            (!isset($param2) || $param2 === '' || !is_string($param2) ) ? $maxLen = $param1 : $maxLen = $param2;
+
+            if (!isset($values) || $values === '') {
+                return null;
```

Comment:
The PHP LIVR library we are extending just returns in cases like this instead of returning null, I think we should do the same.

e.g: https://github.com/umotif-public/php-validator-livr/blob/master/lib/Validator/LIVR/Rules/Common.php